portal-impl/src/com/liferay/portal/service/impl/WebsiteServiceImpl.java

```diff
@@ -63,6 +64,24 @@ public void deleteWebsite(long websiteId) throws PortalException {
 		websiteLocalService.deleteWebsite(website);
 	}
 
+	@Override
+	public Website fetchWebsiteByExternalReferenceCode(
+			String externalReferenceCode, long companyId)
```

Comment:
Sort alphabetically the parameters, so companyId first

Reply:
@smottal if there is ERC being used? Shouldn't it be the first param always?

Reply:
yes, externalReferenceCode in the fetch method goes always first (like in the add method)